src/allotropy/parsers/novabio_flex2/novabio_flex2_structure.py

```diff
@@ -65,25 +69,27 @@ def __lt__(self, other: Any) -> bool:
 @dataclass(frozen=True)
 class Sample:
     identifier: str
-    role_type: str
+    sample_type: str
     measurement_time: str
     batch_identifier: str | None
     analytes: list[Analyte]
-    properties: dict[str, Any]
-
-    @staticmethod
-    def create(series: pd.Series[Any]) -> Sample:
+    cell_counter_properties: dict[str, Any]
+    blood_gas_properties: dict[str, Any]
+    osmolality_properties: dict[str, Any]
+    ph_properties: dict[str, Any]
+    cell_type_processing_method: str | None
+    cell_density_dilution_factor: float | None
+
+    @classmethod
+    def create(cls, series: pd.Series[Any]) -> Sample:
         data = SeriesData(series)
-        properties: dict[str, Any] = {
-            property_name: property_dict["cls"](
-                value=data.get(float, property_dict["col_name"])
-            )
-            for property_name, property_dict in PROPERTY_MAPPINGS.items()
-            if data.get(float, property_dict["col_name"]) is not None
-        }
+        cell_density_dilution = data.get(str, "Cell Density Dilution", "")
```

Comment:
"Cell Density Dilution" Is this useful to be a constant

Reply:
I think for property access strings the answer is no. These are one-time-use and very widespread (we're always having to access columns of dataframes). It would be onerous to make a constant for every one.